.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -1,40 +1,33 @@
 name: Bug Report
 description: Report a bug
 labels: [bug]
-title: '[BUG]: Tell us a clear and concise description of what the bug is'
+title: "[Bug]:"
```

Comment:
I want contributors don't have to delete any text of title

<img width="825" alt="image" src="https://github.com/suspensive/react/assets/61593290/2b82a610-8890-4412-ae2f-b218e6188b9d">